include/copc-lib/hierarchy/key.hpp

```diff
@@ -3,12 +3,24 @@
 
 #include <cstdint>
 #include <functional> // for hash
+#include <limits>
 #include <sstream>
+#include <stdexcept>
 #include <vector>
 
+#include <copc-lib/hierarchy/box.hpp>
+#include <copc-lib/hierarchy/vector3.hpp>
+#include <copc-lib/las/header.hpp>
+
 namespace copc
 {
 
+namespace las
+{
+class LasHeader;
```

Comment:
I don't think we need these?

Reply:
I do on my side, try to remove it and let me know if it works for you.